server/config.ts

```diff
@@ -65,15 +62,6 @@ export default {
       clientId: get('GOVUK_ONE_LOGIN_CLIENT_ID', 'clientId', requiredInProduction),
       privateKey: get('GOVUK_ONE_LOGIN_PRIVATE_KEY', 'privateKey', requiredInProduction),
     },
-    tokenVerification: {
```

Comment:
All token verification stuff removed as this was only used for verifying the _users_ token with HMPPS Auth when that was how they logged in.